src/AppRoutes.tsx

```diff
@@ -13,7 +13,7 @@ const PublicRegistration = lazy(() => import('./pages/public_registration/Public
 const NotFound = lazy(() => import('./pages/errorpages/NotFound'));
 const PrivacyPolicy = lazy(() => import('./pages/infopages/PrivacyPolicy'));
 const ProjectsPage = lazy(() => import('./pages/projects/ProjectsPage'));
-const PublicProfile = lazy(() => import('./pages/research_profile/ResearchProfile'));
+const PublicProfile = lazy(() => import('./pages/research_profile/PublicResearchProfile'));
```

Comment:
Ser det var feil fra før også, men fint om disse har samme navn
```suggestion
const PublicResearchProfile = lazy(() => import('./pages/research_profile/PublicResearchProfile'));
```